exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/masked_modelling.ipynb

```diff
@@ -60,11 +60,11 @@
    "id": "e0ad672a-8b26-467b-8649-0bf95b1efb61",
```

Comment:
<p><s>"As you can see"</s> - I can't really see it, I don't know what the outputs are there. "<s>If you need more details to your output</s>", </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/453/discussion/#comment-2589458084'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>